mrmustard/types.py

```diff
@@ -12,6 +12,10 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+"""A module containing all base type annotations."""
+
+# pylint: disable=unused-wildcard-import,wildcard-import
+
 from typing import *
```

Comment:
Possibly better to import the needed imports here explicitly instead? 🤔

Reply:
I think the idea with the `types` module is for it to be a proxy for `typing`/type annotations. I also thought of removing this wildcard and explicitly importing the needed types; however, when on a different module every time one is required to import a new type then it has to be added to the `types` module import — quite obscure for someone that might not be familiar with Mr Mustard.

I don't know how bad is to bloat the namespace with all the types though. Anyhow, when importing from a module other than `types`, one will import only whatever is required. 

What do you think?

Reply:
Oh, I see. I think this is actually the way we've extended NumPy in PennyLane. I don't have any other suggestions, so I'd say we can keep it like this. 🙂